Makefile

```diff
@@ -28,7 +28,12 @@ VERSION               := $(GIT_TAG)
 endif
 
 # should we build the static files?
+ifneq (,$(filter $@,codegen|lint|test))
+STATIC_FILES          := false
+else
```

Comment:
Do not build static files in the target is one of these

Reply:
nice - I did not know about that